### PR TITLE
Ensure css is minified correctly

### PR DESCRIPTION
**Pull request by twokul**
fixes https://github.com/ember-cli/ember-cli/issues/7755

### CODE REVIEW

File: lib/broccoli/ember-app.js [hunk 1]
```diff
@@ -1599,7 +1600,7 @@ class EmberApp {
         overwrite: true,
       }));
 
-      if (options.minifyCSS.enabled === true) {
+      if (cssMinificationEnabled === true) {
         preprocessedStyles = preprocessMinifyCss(preprocessedStyles, options);
```
Comment (kellyselden) [inline lib/broccoli/ember-app.js]: Don't you want to refactor `preprocessMinifyCss` to expect `options.minifyCSS` up a level? Or is this more of a temp fix?

**Comment by kellyselden** [comment]
Shouldn't you target release for this?

**Comment by twokul** [comment]
@kellyselden I'll cherry pick later and release; issue needs to be fixed on all three: `canary`, `beta` and `release`

**Comment by kellyselden** [comment]
Or you could fix on release, then cascade merge 😉

**Comment by twokul** [comment]
@kellyselden okay 😛

**Comment by twokul** [comment]
superseded via https://github.com/ember-cli/ember-cli/releases/tag/v3.1.2

**Pull request closed without merging**